### PR TITLE
add videos

### PR DESCRIPTION
**Pull request by SashoStoichkov**

### CODE REVIEW

File: content/scripts/video.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+var player;
```
Comment (atodorov) [inline content/scripts/video.js]: Renamed file, not loaded in index.html page

File: content/scripts/video.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+function onYouTubeIframeAPIReady(id) {
+    player = new YT.Player(id);
```
Comment (atodorov) [inline content/scripts/video.js]: See:
https://developers.google.com/youtube/iframe_api_reference#Loading_a_Video_Player

from what I understand we can have 1 player on the page (b/c it is created when the API functions load and call `onYouTubeIframeAPIReady()`. However we should be able to point to different video ID's.

This means we can probably keep the original iframe from the demo video but change `showVideo()` to play a different file for the various buttons. You have to experiment with it.

**Review by atodorov — CHANGES_REQUESTED** [review]

File: content/scripts/video.js [hunk 3]
```diff
@@ -3,12 +3,13 @@ function onYouTubeIframeAPIReady() {
...
 
-function showVideo() {
+function showVideo(videoId) {
     document.getElementById('demoVideo').style.display = "block";
+    player.loadVideoById(videoId);
     player.playVideo();
 }
 
-function hideVideo() {
+function hideVideo(id) {
```
Comment (atodorov) [inline content/scripts/video.js]: `id` parameter not used. Remove it.

**Review by atodorov — CHANGES_REQUESTED** [review]

File: theme/static/style/style.css [hunk 4]
```diff
@@ -413,6 +413,23 @@ pre code {
     cursor: pointer;
 }
 
+/* Seen At Section */
+
+.fosdem-img {
+    width: 200px;
+    height: 200px;
+    background-image: url('../../images/conf/fosdem.png');
```
Comment (atodorov) [inline theme/static/style/style.css]: Use `/images/....` absolute URL

File: theme/static/style/style.css [hunk 5]
```diff
@@ -413,6 +413,23 @@ pre code {
...
+    background-color: inherit;
```
Comment (atodorov) [inline theme/static/style/style.css]: not needed, remove

File: content/pages/index.html [hunk 6]
```diff
@@ -343,14 +343,10 @@ <h3 id="heading">Create IEEE 829 or free text Test Plan</h3>
...
-                    <img style="background-color: inherit" src="/images/conf/fosdem.png" alt="FOSDEM'19 logo" title="FOSDEM Brussels">
-
-                    <div>
-                        <a href="#" title="Watch video" onclick="showVideo('9kOnf5YXquA')">
-                            <i class="fa fa-play fa-3x"></i>
-                        </a>
-                    </div>
+                <div class="fosdem-img" title="FOSDEM Brussels">
+                    <a href="#" title="Watch video" onclick="showVideo('9kOnf5YXquA')">
+                        <i class="fa fa-play fa-2x"></i>
```
Comment (atodorov) [inline content/pages/index.html]: Remove `fa-2x`, let's use a smaller play button here. Larger ones will make the section look busier when you add more of them.

File: theme/static/style/style.css [hunk 7]
```diff
@@ -413,6 +413,23 @@ pre code {
...
+    background-repeat: no-repeat;
+    display: inline-block;
+    float: left;
```
Comment (atodorov) [inline theme/static/style/style.css:424]: This div is out of alignment with the rest of the images. Use `margin-left: 1.5rem;`

**Review by atodorov — CHANGES_REQUESTED** [review]
squash all commits together

File: content/pages/index.html [hunk 8]
```diff
@@ -342,7 +342,13 @@ <h3 id="heading">Create IEEE 829 or free text Test Plan</h3>
                 <img style="background-color: inherit" src="/images/conf/hackconf.png" alt="HackConf'18 logo" title="HackConf Sofia">
                 <img style="background-color: inherit" src="/images/conf/piterpy.png" alt="PiterPy logo" title="PiterPy St. Petersburg">
                 <img style="background-color: inherit" src="/images/conf/openfest.png" alt="OpenFest logo" title="Open Fest Sofia">
-                <img style="background-color: inherit" src="/images/conf/fosdem.png" alt="FOSDEM'19 logo" title="Open Source Test Management stand at FOSDEM">
+
+                <div class="fosdem-img" title="FOSDEM Brussels">
+                    <a href="#" title="Watch video" onclick="showVideo('9kOnf5YXquA')">
+                        <i class="fa fa-play fa-x"></i>
```
Comment (atodorov) [inline content/pages/index.html]: There is no `fa-x` class.

**Review by atodorov — CHANGES_REQUESTED** [review]

File: content/pages/index.html [hunk 9]
```diff
@@ -342,7 +342,13 @@ <h3 id="heading">Create IEEE 829 or free text Test Plan</h3>
                 <img style="background-color: inherit" src="/images/conf/hackconf.png" alt="HackConf'18 logo" title="HackConf Sofia">
                 <img style="background-color: inherit" src="/images/conf/piterpy.png" alt="PiterPy logo" title="PiterPy St. Petersburg">
                 <img style="background-color: inherit" src="/images/conf/openfest.png" alt="OpenFest logo" title="Open Fest Sofia">
-                <img style="background-color: inherit" src="/images/conf/fosdem.png" alt="FOSDEM'19 logo" title="Open Source Test Management stand at FOSDEM">
+
+                <div class="fosdem-img" title="FOSDEM Brussels">
+                    <a href="#" title="Watch Fosdem video" onclick="showVideo('9kOnf5YXquA')">
+                        <i class="fa fa-play fa-lg"></i>
```
Comment (atodorov) [inline content/pages/index.html]: Remove `fa-lg`, see previous comment.

**Review by atodorov — CHANGES_REQUESTED** [review]
almost there

**Comment by SashoStoichkov** [comment]
I thing it's ready :)

**Review by atodorov — APPROVED** [review]

**Pull request merged**